### ISSUE TITLE
Leapmotion when UE4 game doesn't have focus

### ISSUE THREAD

**Issue by Ozzadar** [getnamo/leap-ue4#12]
Hi,

I've been doing some experimentation and I have a couple of questions on some minor annoyances:

1- Is there a way to keep receiving leapmotion data when the UE4 game (or editor) doesnt have focus? Right now as soon as you alt tab the service is 'paused'

2- In a packaged build, when the hands are not seen by leap the hands are not hidden in game (leaving me with the hands obscuring the camera) -- Is this a bug or is there a quick fix that I can apply for this?

Thanks,
-Ozzadar

**Comment by Ozzadar** [comment]
I've solved issue #1, found the EnableBackgroundTracking on leapcontroller; though I can't get it to work properly on BeginPlay or in Constructor. I have to stick it in my Tick function for it to work as intended.

Thoughts?

**Comment by getnamo** [comment]
interesting on 2, if you bring your hands in view and out of the view again do they hide? There may be a toggle missing on the character bp

**Comment by Ozzadar** [comment]
Nope, they stay visible no matter what I do. 
It's quite perplexing. I've taken a look through the blueprints and can't seem to find the root cause. =/

**Comment by getnamo** [comment]
and this only happens when the game is packaged?

**Comment by Ozzadar** [comment]
Yeah, I just built a packaged game with the base blueprint and experienced the same behaviour (incase it was something wonky going on with my custom character)

Definitely something fishy happening =/

**Comment by Ozzadar** [comment]
Oh, on a related note(to #1); it seems that leapmotion background tracking crashes unreal engine after a few minutes. 

Read access violation error of some sort though I dont have the .pdb loaded so I cant track it down perfectly. I'll be attempting to check that out for you later (may even submit a change if I can fix it)

**Comment by getnamo** [comment]
are you using the in-engine plugin or a project plugin using latest updates from this repo? Also what engine version?

**Comment by Ozzadar** [comment]
4.12.4 and using the plugin included with the engine. Is that one woefully out of date?

**Comment by getnamo** [comment]
It isn't and it is in fact probably the most stable release, but I have done some changes to the repo here which may or may not alleviate this problem. You can try it out by just downloading the repo, dragging the Plugins folder into your project. Let me know if that changes anything.
